semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -263,19 +270,32 @@ def parse_id(attribute):
             # TODO: If validation fails later, this will crash when trying to access position data
             # since a list, contrary to a CommentedSeq, does not have position data
             examples = [examples]
-        if is_simple_type and attr_type not in (
-            "boolean",
-            "boolean[]",
-            "int",
-            "int[]",
-            "double",
-            "double[]",
+        if is_template_type or (
+            is_simple_type
+            and attr_type
+            not in (
+                "boolean",
+                "boolean[]",
+                "int",
+                "int[]",
+                "double",
+                "double[]",
+            )
         ):
             if not examples:
```

Comment:
Examples are a good point. You'd want to show the key in the example for templates as well. But that can be done i a follow-up / is a nice-to-have

Reply:
> But that can be done i a follow-up / is a nice-to-have

Agree